src/domain/Layouts/Inline/Inline.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import { Props, Variables } from '../../../common'
+import { StyledInlineElement, StyledInlineGroup, StyledInlineGroupWrapper } from './style'
+
+interface IInlineProps {
```

Comment:
should we have collapseBelow props too? It is highly used in out system. We are using the grid layout for full-width mobile right now.

Reply:
I'm not sure. CollapseBelow doesnt work in cases like goal template (unless we don't want to do that anymore...)

Reply:
We will support flex grow and collapse below